src/main.rs

```diff
@@ -119,40 +137,13 @@ async fn handle_streamer_message(
         contracts,
     )
     .await?;
-    Ok(streamer_message.block.header.height)
-}
 
-fn init_tracing() -> DefaultSubcriberGuard {
-    let mut env_filter = EnvFilter::new("near_lake_framework=info,indexer_events=info");
-
-    if let Ok(rust_log) = env::var("RUST_LOG") {
-        if !rust_log.is_empty() {
-            for directive in rust_log.split(',').filter_map(|s| match s.parse() {
-                Ok(directive) => Some(directive),
-                Err(err) => {
-                    tracing::warn!(
-                        target: crate::LOGGING_PREFIX,
-                        "Ignoring directive `{}`: {}",
-                        s,
-                        err
-                    );
-                    None
-                }
-            }) {
-                env_filter = env_filter.add_directive(directive);
-            }
-        }
-    }
+    let time_since_the_epoch = SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .expect("Something went wrong.");
 
-    let (non_blocking_writer, _guard) = tracing_appender::non_blocking(std::io::stderr());
-
-    let subscriber = tracing_subscriber::FmtSubscriber::builder()
-        .with_env_filter(env_filter)
-        .with_writer(non_blocking_writer)
-        .finish();
-
-    DefaultSubcriberGuard {
-        subscriber_guard: tracing::subscriber::set_default(subscriber),
-        writer_guard: _guard,
-    }
+    LATEST_BLOCK_HEIGHT.set(streamer_message.block.header.height.try_into().unwrap());
```

Comment:
collect metrics

Reply:
I believe @khorolets and even in nearcore, they opted to use IntGuage. 

https://github.com/khorolets/near-lake-pulse/blob/52033d1cc128d231a99b26386e380c5a44dfd5d4/src/main.rs#L23

Reply:
I've lost this thread, sorry.

> I believe @khorolets and even in nearcore, they opted to use IntGuage.

No matter who uses this type, they anyway can be wrong. 

I'm against adding f64 here, mostly because it looks confusing.

I'm OK with the current solution (proper error handling + comment why we do it this way).
Even better, we can create the issue in rust-prometheus asking to support u64, u128, and leave the link to the issue in the comments in our project. With this, we can track the progress and fix this when the library will be updated.

Reply:
Created an issue for this on `rust-prometheus` -> https://github.com/tikv/rust-prometheus/issues/470